tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ts

```diff
@@ -26,20 +26,25 @@ import {
   getScaleRangeFromDomDim,
 } from './chart_view_utils';
 import {
+  filterTicksByVisibility,
   getStandardTicks,
   getTicksForLinearScale,
   getTicksForTemporalScale,
   MajorTick,
   MinorTick,
 } from './line_chart_axis_utils';
 
+const AXIS_FONT = '11px Roboto, sans-serif';
+
 @Component({
   selector: 'line-chart-axis',
   templateUrl: 'line_chart_axis_view.ng.html',
   styleUrls: ['line_chart_axis_view.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class LineChartAxisComponent {
+  readonly axisFont = AXIS_FONT;
```

Comment:
is this variable used anywhere? If you are not going to change it, line 107 is good enough?

Reply:
Thanks, it was a mistake.